src/server/models/match.model.js

```diff
@@ -15,7 +15,7 @@ let matchSchema = new Schema({
     default: Date.now(),
   },
   response_date: Date,
-  //status of match {pending:0, accepted:1, rejected:2}
+  //status of match {pending:2, accepted:1, rejected:0}
```

Comment:
This mistake made only at comment. All code is based on 
{pending:2, accepted:1, rejected:0}